web/client/components/misc/Thumbnail.jsx

```diff
@@ -89,20 +89,21 @@ const Thumbnail = forwardRef(({
 
     const handleDrop = (files) => {
         const imageType = files?.[0]?.type;
+        const imageSize = files?.[0]?.size;
```

Comment:
I would propose a different approach, a new prop called `checkOriginalFileSize`, if true we can check directly here without performing the resize of the image, something like this:

Note: checkOriginalFileSize should be set true only for the background dialog

```js
const imageSize = files?.[0]?.size;
if (checkOriginalFileSize && imageSize >= maxFileSize) {
    return onError(['SIZE'], files);
}
```

This should restore previous behaviour and it should not affect other thumbnails that are not using the original image size limit.